config/dependencies.yml

```diff
@@ -75,4 +75,7 @@ googleapis_common_protos:
 nodeModules:
   arguejs:
     version: '0.2.3'
-
+  extend:
+    version: '3.0.0'
+  lodash:
+    version: '4.6.0'
```

Comment:
I think we need a dependency like

google-proto-files:
       version:  '0.7.0'